micropub.php

```diff
@@ -70,6 +70,9 @@ class Micropub_Plugin {
 	public static function init() {
 		$cls = get_called_class();
 
+		// Admin Init
```

Comment:
very minor nit: probably remove this comment, it doesn't add anything beyond reading the code itself. :P